internal/repository/repository_test.go

```diff
@@ -0,0 +1,370 @@
+package repository
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/VTerenya/employees/internal"
+	errs "github.com/VTerenya/employees/internal/errors"
+	"github.com/google/uuid"
+	"github.com/shopspring/decimal"
+)
+
+var (
+	data        *Database   //nolint: gochecknoglobals
+	repos       *Repository //nolint: gochecknoglobals
+	positionIDs []string    //nolint: gochecknoglobals
+	employeeIDs []string    //nolint: gochecknoglobals
+)
```

Comment:
You can use only one comment before var block

Reply:
+

Reply:
It doesn't work.